examples/data-objects/table-document/src/cellrange.ts

```diff
@@ -47,6 +47,9 @@ export function parseRange(range: string) {
 	return { minRow, minCol, maxRow, maxCol };
 }
 
+/**
+ * @alpha
+ */
 export class CellRange {
```

Comment:
Adding tags without api-extractor? Is this just a comment leveraging tags or ??

Reply:
Since someone had (mostly) tagged this type, I oscillated on adding api-extractor vs. removing.

I eventually decided adding was silly, since it's an example with the main types marked as deprecated.